### PR TITLE
pymelts recipe added

### PR DESCRIPTION
**Pull request by msghiorso**
pymelts python package. Builds an extension package (in C) and associated python bindings.

### CODE REVIEW

**Comment by msghiorso** [comment]
First build completed.

File: recipes/pymelts/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  build:
+    - python
+    - setuptools
+    - toolchain
+  run:
+    - python
+
+test:
+  commands:
+    - echo "Tests completed"
```
Comment (scopatz) [inline recipes/pymelts/meta.yaml]: This is not a comprehensive enough test. Please include at least simple existence tests: http://conda-forge.github.io/docs/testing.html#simple-existence-tests

File: recipes/pymelts/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+about:
+  home: http://gitlab.com/enki-portal/xMelts
+  license: BSD
```
Comment (scopatz) [inline recipes/pymelts/meta.yaml]: Please add the `licesne_file` entry if possible.

**Comment by scopatz** [comment]
Please enable CircleCI for your fork of staged recipes, so that the linux build will run for this PR. Thanks!

**Comment by synapticarbors** [comment]
@scopatz I've seen this comment on enabling CircleCI for the fork in several PRs, but is there documentation/notes on how to do this?

**Comment by scopatz** [comment]
This is the only documentation that Circle has: https://circleci.com/docs/getting-started/#add-and-follow-more-projects Mostly it involves going to https://circleci.com/add-projects and finding the project / fork name.

**Comment by scopatz** [comment]
@synapticarbors documentation PR at conda-forge/conda-forge.github.io#299

File: recipes/pymelts/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    - toolchain
+  run:
+    - python
+
+test:
+  commands:
+    - echo "Tests completed"
+
+about:
+  home: http://gitlab.com/enki-portal/xMelts
```
Comment (jakirkham) [inline recipes/pymelts/meta.yaml]: This URL does not appear to be accessible. 😕

File: recipes/pymelts/meta.yaml (hunk 2 above)
Comment (jakirkham) [inline recipes/pymelts/meta.yaml]: Also please specify the number of clauses if BSD (e.g. `BSD 3-Clause` for the 3 clause version).

File: recipes/pymelts/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://repository.ofm-research.org/packages/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ checksum }}
+
+build:
+  number: 1
```
Comment (jakirkham) [inline recipes/pymelts/meta.yaml]: This needs some sort of installation step. Please see the example included in this repo for an idea of how to do this.

xref: https://github.com/conda-forge/staged-recipes/blob/a172ea2b0031eb31d79476efa7fe731e566bc204/recipes/example/meta.yaml#L26

Comment (jakirkham) [inline recipes/pymelts/meta.yaml]: Also please set the build number to `0`.

**Comment by jakirkham** [comment]
Toggling to try and get CircleCI back.

**Comment by jakirkham** [comment]
...and CircleCI joins the party. 🎉

**Comment by jakirkham** [comment]
Any thoughts on these other points, @msghiorso.

**Comment by msghiorso** [comment]
I have updated the meta.yaml file for this project and will push.  I do not understand what to do re the CircleCI issue.  Links to documentation seem to be broken.

File: .gitignore [hunk 5]
```diff
@@ -1 +1,3 @@
 build_artefacts
+/.DS_Store
```
Comment (synapticarbors) [inline .gitignore]: While I think adding the `.DS_Store` stuff is harmless, I don't think a recipe should necessarily be making changes to the code external to the submitted recipe.

Comment (msghiorso) [inline .gitignore]: Okay.  So, what should I have done?

**Comment by msghiorso** [comment]
On suggestion to add an installation line to the build section of meta.yaml, I followed the example recipe and added
setup.py install --single-version-externally-managed --record record.txt
but the option --single-version-externally-managed is apparently unsupported.  Should this just be
setup.py install
?

**Pull request closed without merging**